user-guide/content/docs/faq.md

```diff
@@ -5,18 +5,18 @@ title: FAQ
 
 {{<expand "Why aren't my unused addresses imported when I restore a wallet?" "...">}}
 
-This is by vertue of the blockchain. An unused address is by definition unused. Meaning that is doesn't exist on the chain and only exists locally, in the context of the software that has generated it. Different softwares may use different rules to generate addresses. For example in the past, _cardano-sl_ wallets used a method called random derivation where addresses will be created from a root seed and a random index that is stored with the address itself. Because indexes are random, it is not possible to restore randomly generated addresses that haven't been used on chain! 
+This is by vertue of the blockchain. An unused address is by definition unused. Meaning that is doesn't exist on the chain and only exists locally, in the context of the software that has generated it. Different softwares may use different rules to generate addresses. For example in the past, _cardano-sl_ wallets used a method called random derivation where addresses were created from a root seed and a random index stored within the address itself. Because these indexes were random, it was not possible to restore randomly generated addresses that haven't been used on chain yet! 
```

Comment:
```suggestion
This is by virtue of the blockchain. An unused address is by definition unused. Meaning that is doesn't exist on the chain and only exists locally, in the context of the software that has generated it. Different software may use different rules to generate addresses. For example in the past, _cardano-sl_ wallets used a method called random derivation where addresses were created from a root seed and a random index stored within the address itself. Because these indexes were random, it was not possible to restore randomly generated addresses which hadn't been used on chain yet! 
```